src/main/java/smartthings/ratpack/sqs/DefaultSqsService.java

```diff
@@ -35,7 +37,8 @@ public DefaultSqsService(
     @Override
     public void onStart(StartEvent event) throws Exception {
         if (config.isEnabled()) {
-            this.sqs = sqsConfigService.getAmazonSQSAsync(config.getRegion());
+            sqs = sqsConfigService.getAmazonSQSAsync(config.getRegion());
+            Optional.ofNullable(config.getEndpoint()).ifPresent(sqs::setEndpoint);
```

Comment:
@jeff-blaisdell I agree in that there are way to many config options to cover all use cases with config parameters.  Setting up default config and allowing apps to override instantiation of the underlying client seems like an ideal approach.  The optional endpoint parameter that @timic proposed here seems ok.  I originally thought this was the actual queue URL but after re-reading AWS docs the use case for setting this is a clearer and seems like a useful config value to include.